gadgets/trace_bind/gadget.yaml

```diff
@@ -3,53 +3,63 @@ description: Trace socket bindings
 homepageURL: https://inspektor-gadget.io/
 documentationURL: https://inspektor-gadget.io/docs
 sourceURL: https://github.com/inspektor-gadget/inspektor-gadget/
+datasources:
+  bind:
+    fields:
+      timestamp_raw:
+        annotations:
+          columns.hidden: true
+      timestamp:
+        annotations:
+          template: timestamp
+      mount_ns_id:
+        annotations:
+          template: ns
+      addr:
+        annotations:
+          description: Address the socket is bound to
+          columns.width: 32
```

Comment:
Yeah, but I think we should handle it with a template later on too. (I won't handle in this PR because the endpoints behave a little bit different and the template wasn't working for me).